db/schema.rb

```diff
@@ -12,57 +10,57 @@
+  enable_extension "plpgsql"
 
-  create_table 'good_job_processes', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
-    t.datetime 'created_at', precision: 6, null: false
-    t.datetime 'updated_at', precision: 6, null: false
-    t.jsonb 'state'
+  create_table "good_job_processes", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.jsonb "state"
   end
 
-  create_table 'good_jobs', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
-    t.text 'queue_name'
-    t.integer 'priority'
-    t.jsonb 'serialized_params'
-    t.datetime 'scheduled_at'
-    t.datetime 'performed_at'
-    t.datetime 'finished_at'
-    t.text 'error'
-    t.datetime 'created_at', precision: 6, null: false
-    t.datetime 'updated_at', precision: 6, null: false
-    t.uuid 'active_job_id'
-    t.text 'concurrency_key'
-    t.text 'cron_key'
-    t.uuid 'retried_good_job_id'
-    t.datetime 'cron_at'
-    t.index %w[active_job_id created_at], name: 'index_good_jobs_on_active_job_id_and_created_at'
-    t.index ['concurrency_key'], name: 'index_good_jobs_on_concurrency_key_when_unfinished',
-                                 where: '(finished_at IS NULL)'
-    t.index %w[cron_key created_at], name: 'index_good_jobs_on_cron_key_and_created_at'
-    t.index %w[cron_key cron_at], name: 'index_good_jobs_on_cron_key_and_cron_at', unique: true
-    t.index %w[queue_name scheduled_at], name: 'index_good_jobs_on_queue_name_and_scheduled_at',
-                                         where: '(finished_at IS NULL)'
-    t.index ['scheduled_at'], name: 'index_good_jobs_on_scheduled_at', where: '(finished_at IS NULL)'
+  create_table "good_jobs", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.text "queue_name"
+    t.integer "priority"
+    t.jsonb "serialized_params"
+    t.datetime "scheduled_at"
+    t.datetime "performed_at"
+    t.datetime "finished_at"
+    t.text "error"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.uuid "active_job_id"
+    t.text "concurrency_key"
+    t.text "cron_key"
+    t.uuid "retried_good_job_id"
+    t.datetime "cron_at"
+    t.index ["active_job_id", "created_at"], name: "index_good_jobs_on_active_job_id_and_created_at"
+    t.index ["concurrency_key"], name: "index_good_jobs_on_concurrency_key_when_unfinished", where: "(finished_at IS NULL)"
+    t.index ["cron_key", "created_at"], name: "index_good_jobs_on_cron_key_and_created_at"
+    t.index ["cron_key", "cron_at"], name: "index_good_jobs_on_cron_key_and_cron_at", unique: true
+    t.index ["queue_name", "scheduled_at"], name: "index_good_jobs_on_queue_name_and_scheduled_at", where: "(finished_at IS NULL)"
+    t.index ["scheduled_at"], name: "index_good_jobs_on_scheduled_at", where: "(finished_at IS NULL)"
```

Comment:
Is this change intentional?

Reply:
No this was automatic regeneration of the schema during database recreation